### PR TITLE
This includes all operation for singly linkedlist

### PR DESCRIPTION
**Pull request by anjisnu07**

### CODE REVIEW

File: Linked_list_all_operations_in_all.c [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+void insert_from_begining(){
+	struct node *ptr;
+	ptr=(struct node *)malloc(sizeof(struct node));
+	printf("Enter data you wanna add:\n");
+	scanf("%d",&ptr->data);
+	printf("You inserted:%d\n",ptr->data);
+	ptr->next=NULL;
+	ptr->next=head;
+	head=ptr;
+//	while(ptr->next !=NULL){
```
Comment (mhmdsami) [inline Linked_list_all_operations_in_all.c]: Why do you have this part commented?

File: Linked_list_all_operations_in_all.c [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+		case 3:insert_from_begining();
+		break;
+		case 4:display();
+		break;
+		case 5:del_from_end();
+		break;
+		case 6:del_from_begining();
+		break;
+		case 7:del_from_A_certain_position();
+		break;
```
Comment (mhmdsami) [inline Linked_list_all_operations_in_all.c]: Can you also add an option to exit the program?

**Review by mhmdsami — CHANGES_REQUESTED** [review]
The program is not formatted properly, please run this through Prettier once. 

Also look into [Contributing Guide](https://github.com/mhmdsami/dsa#handshake-contributing-guide) and rename the file and put it into the correct folder.

File: Linked_list_all_operations_in_all.c (hunk 1 above)
Comment (anjisnu07) [reply Linked_list_all_operations_in_all.c]: cause we actually don't need to traverse the list

**Review by anjisnu07 — COMMENTED** [review]
Now check if its okay!

**Comment by mhmdsami** [comment]
@anjisnu07 It is not properly formatted yet

**Comment by anjisnu07** [comment]
@mhmdsami I'm not getting the format of folder btw!! Code is alright ig..and changed everything accordingly.. if you can merge it like this then it's okay.

**Comment by mhmdsami** [comment]
@anjisnu07 You can add this to `Linked_List/Exercise` and name it `single-linked-list-all-operation.c`!, are you sure you have used Prettier to format the code, it does not seem like it!

**Pull request closed without merging**